projects/hello_world/src/hello_world.c

```diff
@@ -0,0 +1,35 @@
+//
+// Created by henry on 2021-01-09.
+//
+#include <stdint.h>
+
+#include "interrupt.h"
+#include "log.h"
+#include "soft_timer.h"
+
+typedef struct Counters {
+  uint8_t counter_a;
+  uint8_t counter_b;
+} Counters;
+void callback(SoftTimerId id, void *context) {
```

Comment:
Since this is a private function, our [coding standards](https://uwmidsun.atlassian.net/wiki/spaces/ELEC/pages/9306218/Coding+Standards) say that it should be declared `static` and prefixed with `prv_`. The `static` means that it can't be used by other files.